exercise-core/src/main/java/com/examples/attsw/exercise/core/model/Employee.java

```diff
@@ -2,14 +2,25 @@
 
 public class Employee {
 
+	private String name;
```

Comment:
Let's take the opportunity to put "id" as the first field; toString should be changed accordingly